frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -27,7 +27,6 @@ local MODE = {
     book_time_to_read = 6,
     chapter_time_to_read = 7,
     frontlight = 8,
-    mem_usage = 9,
```

Comment:
Until our memleak issue is fixed, a lot of the users will still rely on this footer info as a quick restart reminder unfortunately.